### ISSUE TITLE
Small question about spines

### ISSUE THREAD

**Issue by ahartikainen** [arviz-devs/arviz#558]
Hi,

just a quick question on which method should be used when removing spines

```
for loc, spine in ax.spines.items():
    if loc in ["left", "right"]:
        spine.set_color("none")  # don't draw spine
```
or 

```
for loc, spine in ax.spines.items():
    if loc in ["left", "right"]:
        spine.set_visible(False)  # don't draw spine
```

**Comment by aloctavodia** [comment]
I vote for the second, because the intention is _more_ explicit.

**Comment by canyon289** [comment]
I vote for second, same reasoning as @aloctavodia.

**Comment by ahartikainen** [comment]
Fixed by #559
